File: four-days-java-web/day-four/tcc/src/main/java/com/invillia/tcc/domain/request/ContactRequest.java
package com.invillia.tcc.domain.request;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;
import lombok.Data;

@Data
public class ContactRequest {

    @NotBlank
    private String name;

    @Email
    @NotBlank
    private String email;

}

File: my-integration-test-timeline/future/src/main/kotlin/br/com/leonardoferreira/future/domain/request/ContactRequest.kt
package br.com.leonardoferreira.future.domain.request

import javax.validation.constraints.Email
import javax.validation.constraints.NotBlank

data class ContactRequest(

    @field:NotBlank
    val name: String,

    @field:Email
    @field:NotBlank
    val email: String

)
